pkg/runtime/runtime.go

```diff
@@ -544,6 +546,7 @@ func (a *DaprRuntime) initRuntime(opts *runtimeOpts) error {
 			OutputBindings:  a.outputBindings,
 			SecretStores:    a.secretStores,
 			PubSubs:         a.pubSubs,
+			WorkFlows:       a.workFlows,
```

Comment:
`Workflows`?

Reply:
"workFlows" follows the same naming convention as the items above it